### PR TITLE
Flatten logo_$_ids, font_$_ids and reverse fields to remove template fields.

### PR DESCRIPTION
**Pull request by reiterl**
Update model, tests, action

### CODE REVIEW

**Review by r-peschke — CHANGES_REQUESTED** [review]
Needs discussion

File: global/meta/models.yml [hunk 1]
```diff
@@ -423,6 +423,55 @@ meeting_user:
     to: chat_message/meeting_user_id
     restriction_mode: A
 
+meeting_logo_mediafile:
+  id:
```
Comment (r-peschke) [inline global/meta/models.yml:427]: All fields required

**Review by r-peschke — CHANGES_REQUESTED** [review]
Just some required, apart from that the models.yml seems okay

**Comment by jsangmeister** [comment]
Didn't we say that we did not want an extra model for this to reduce the overhead being introduced with this change?

**Comment by r-peschke** [comment]
> Didn't we say that we did not want an extra model for this to reduce the overhead being introduced with this change?

This way is faster to implement, better to understand, easier to change in client, etc.

**Comment by jsangmeister** [comment]
> This way is faster to implement

No, instead of just accessing a single field (like before) you know have to fetch a new model and access the field there, which is more overhead code-wise. Also, the "flatten" approach was already implemented by @reiterl in the first commit.

> better to understand

I think a direct relation (or multiple direct relations, for that matter) are easier to understand than a new collection, but I suppose that's a matter of opinion.

> easier to change in client, etc.

I'm pretty sure it's easier to remove a `$` from the existing code than to rewrite it to access a new collection.

I'm against this change and I don't like the way how decisions made in the "big round" are just thrown over board...

File: tests/system/action/meeting/test_clone.py [hunk 2]
```diff
@@ -38,8 +38,6 @@ def setUp(self) -> None:
                 "group_ids": [1, 2],
                 "motion_state_ids": [1],
                 "motion_workflow_ids": [1],
-                "logo_$_id": None,
```
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/system/action/meeting/test_clone.py:41]: Add a test for the logo and font id to clone.

File: tests/system/action/meeting/test_clone.py [hunk 3]
```diff
@@ -606,8 +602,6 @@ def test_clone_with_mediafile(self) -> None:
                 "mediafile/1": {
                     "owner_id": "meeting/1",
                     "attachment_ids": [],
-                    "used_as_font_$_in_meeting_id": [],
```
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/system/action/meeting/test_clone.py:609]: See above

File: tests/system/action/meeting/test_import.py [hunk 4]
```diff
@@ -438,8 +436,6 @@ def get_mediafile_data(
             "list_of_speakers_id": None,
             "projection_ids": [],
             "attachment_ids": [],
-            "used_as_logo_$_in_meeting_id": [],
```
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/system/action/meeting/test_import.py:441]: There is already the logo_dollar_id test, I add a font test.

File: global/meta/models.yml (hunk 1 above)
Comment (reiterl) [reply global/meta/models.yml]: Outdated.

**Review by r-peschke — CHANGES_REQUESTED** [review]
Where is the check for the field names/enums?

File: openslides_backend/presenter/check_mediafile_id.py [hunk 5]
```diff
@@ -104,10 +118,30 @@ def check_permissions(
         #    or used_as_font_$_in_meeting_id is not empty)
         can_see_meeting = self.check_can_see_meeting(meeting)
         if can_see_meeting:
-            if mediafile.get("used_as_logo_$_in_meeting_id") or mediafile.get(
-                "used_as_font_$_in_meeting_id"
+            for field_part in (
+                "projector_main",
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/presenter/check_mediafile_id.py]: Add two tuple to the meeting model. Use the code later.

File: openslides_backend/presenter/check_mediafile_id.py [hunk 6]
```diff
@@ -54,8 +54,22 @@ def get_result(self) -> Any:
                     "owner_id",
                     "token",
                     "mimetype",
-                    "used_as_logo_$_in_meeting_id",
-                    "used_as_font_$_in_meeting_id",
+                    "used_as_logo_projector_main_in_meeting_id",
```
Comment (r-peschke) [inline openslides_backend/presenter/check_mediafile_id.py:57]: Please add this fields in a generic way from the new enums.

**Review by r-peschke — CHANGES_REQUESTED** [review]
Just one small thing, see above. After changing you can merge.

File: openslides_backend/presenter/check_mediafile_id.py (hunk 6 above)
Comment (reiterl) [reply openslides_backend/presenter/check_mediafile_id.py]: Done.

**Review by r-peschke — APPROVED** [review]

**Pull request merged**